lib/cli.rb

```diff
@@ -176,7 +176,7 @@ def gthb_context_suffix_option(parser)
 
     # class InstallOptions
     class InstallOptions
-      attr_accessor :platform, :force, :skip_client
+      attr_accessor :platform, :force, :skip_client, :drivers
```

Comment:
What about having an empty array as the default value for `drivers` by default? That saves some code later, and sounds more semantically correct for me.

Reply:
good idea. Fixed